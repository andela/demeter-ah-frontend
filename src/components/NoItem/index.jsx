import React, { Fragment } from 'react';

const NoItem = ({ message }) => (
  <Fragment>
    <div className="w-full flex h-full justify-center items-center">
      <p className="text-center font-semibold text-lg opacity-25">{message}</p>
    </div>
  </Fragment>
);

export default NoItem;
