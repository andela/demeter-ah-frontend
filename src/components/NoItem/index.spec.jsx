import React from 'react';
import { shallow } from 'enzyme';
import NoItem from './index';

describe('NoItem Component', () => {
  it('Should render without errors', () => {
    const component = shallow(<NoItem />);
    expect(component).toMatchSnapshot();
  });
});
