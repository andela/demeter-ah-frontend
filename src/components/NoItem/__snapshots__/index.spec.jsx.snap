// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoItem Component Should render without errors 1`] = `
<Fragment>
  <div
    className="w-full flex h-full justify-center items-center"
  >
    <p
      className="text-center font-semibold text-lg opacity-25"
    />
  </div>
</Fragment>
`;
